### ISSUE TITLE
Separate Team 900 and Team 2.4 targets in configurator

### ISSUE THREAD

**Issue by jklong03** [ExpressLRS/ExpressLRS-Configurator#138]
Perhaps, as a way to help prevent the selection of incorrect targets, what if an Initial Pull-down option at the top of the screen was presented that had the user select between 900 MHz and 2.4 GHz families of targets?

Then, once that selection is done by the user, only targets from the correct team would be presented in the subsequent pulldowns.

**Comment by jurgelenas** [comment]
Which configurator version you are using? Targets are separated into 2.4 Ghz and 900 Mhz / DIY categories for a while. You get a categories from both teams on the same list though

**Comment by jklong03** [comment]
It's not that big of a deal, just something I thought would be nice to have for future enhancement and to add even more to the "polished look" that nubes seem to judge systems on. FYI I have been with ELRS since April at least Configurator 0.4.14 (oldest version I still have on my drive) currently on V 1.2
![Screenshot 2021-11-21 01-34-35](https://user-images.githubusercontent.com/110277/142756863-3d9573e3-c709-4ec7-b4a1-f5284cffce0d.png)
.
![Screenshot 2021-11-21 01-45-17](https://user-images.githubusercontent.com/110277/142756869-e39d4c09-7648-4400-9eb0-d5781c278d3f.png)
5

**Comment by brandonrc** [comment]
![image](https://user-images.githubusercontent.com/25802530/142889105-ffbf9619-8c4a-4648-8445-8271ed5573d2.png)
I think an option like this would be nice. 

Add some radio buttons to filter out all targets based on team 2.4/ team 900 before any other filters.
